source/deployment/bulk-loading-data-format.rst

```diff
@@ -366,6 +367,13 @@ Fields of the User object
       <td align="center" valign="middle">No</td>
     </tr>
     <tr class="row-odd">
+      <td valign="middle">password</td>
+      <td valign="middle">string</td>
+      <td>A password for the user. If not present, the bulk loader generates a password.</td>
```

Comment:
There are two other things we should note here:
1. The password field (or autogenerated password if the field is omitted) is only applicable if `auth_service` is empty (i.e. Password Authentication is the selected auth service).
2. `auth_data` and `password` are mutually exclusive. You can only use one in a given `User`.